index.html

```diff
@@ -107,6 +107,19 @@ <h4>Dynawo</h4>
                         </div>
                     </div>
                 </div>
+                <div class="col-md-6 col-lg-4 portfolio-item filter-app">
+                    <div class="portfolio-wrap">
+                        <img src="/assets/img/portfolio/emsys_grid_logo_bigger.png" class="img-fluid" alt="emsys">
+                        <div class="portfolio-info">
+                            <h4>emsys grid</h4>
+                            <p>Modeling and exporting forecasted network state in CGMES format. CGMES export validation according to ENTSOE norms & operator inputs</p>
+                            <div class="portfolio-links">
+                                <a href="https://www.emsysgrid.com" title="More Details" target="_blank"><i class="bx bx-link"></i></a>
+                                <a href="" title="GitHub" target="_blank"><i class="bx bxl-github"></i></a>
```

Comment:
This is the little GitHub icon right? Is it necessary to keep it if it does not lead anywhere? I would just put the link to the website maybe.

Reply:
Yes, the github link is not available yet. I can put the website link for the moment!

Reply:
thanks that will do , the github link is in process